File: help/eng_gb/exercicios_da_ficha_4/Ficha04_014/Ficha04_014.java
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

public class Ficha04_014 {

    //Main Function
    public static void main( String[] args) {
        System.out.print("Truth table of OR\n" );
        boolean var = true || true ;
        System.out.print("\n" + true + "\t||\t" + true + "\t= " + var );
        var = true || false ;
        System.out.print("\n" + true + "\t||\t" + false + "\t\t= " + var );
        var = false || true ;
        System.out.print("\n" + false + "\t\t||\t" + true + "\t= " + var );
        System.out.print("\n" + false + "\t\t||\t" + false + "\t\t= " + ( false || false ) );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:28

File: help/eng_gb/exercicios_da_ficha_4/Ficha04_014/Ficha04_014.cpp
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha04_014 {

    public:
        int main() {
            cout << "Truth table of OR\n";
            int var  = true || true ;
            cout << "\n" << true  << "\t||\t" << true  << "\t= " << var ;
            var = true || false ;
            cout << "\n" << true  << "\t||\t" << false  << "\t\t= " << var ;
            var = false || true ;
            cout << "\n" << false  << "\t\t||\t" << true  << "\t= " << var ;
            cout << "\n" << false  << "\t\t||\t" << false  << "\t\t= " <<( ( false || false )  );
            return 0;
        }
};// end of class Ficha04_014
// Execute main function
int main(){
    Ficha04_014 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:28
